### PR TITLE
use python logger instead of print statements for the main cjdns file

### PR DESCRIPTION
**Pull request by thefinn93**

### CODE REVIEW

**Comment by thefinn93** [comment]
This will close #4

File: cjdns/cjdns.py [hunk 1]
```diff
@@ -113,19 +118,18 @@ def _receiverThread(session):
             try:
                 benc = bdecode(data)
             except (KeyError, ValueError):
-                print("error decoding [" + data + "]")
+                logger.warning("error decoding [%s]", data)
```
Comment (kyrias) [inline cjdns/cjdns.py]: Shouldn't this be a `logger.error`?

Comment (thefinn93) [reply cjdns/cjdns.py]: eh proly. I wasn't sure. I'll update it

**Review by kyrias — APPROVED** [review]
LGTM

**Pull request merged**